ci/create_database.py

```diff
@@ -89,7 +89,7 @@ async def create_user_if_doesnt_exist(admin_or_user, mysql_username, mysql_passw
             allowed_operations = 'ALL'
         else:
             assert admin_or_user == 'user'
-            allowed_operations = 'SELECT, INSERT, UPDATE, DELETE, EXECUTE'
+            allowed_operations = 'SELECT, INSERT, UPDATE, DELETE, EXECUTE, CREATE TEMPORARY TABLES'
```

Comment:
I think if would be fine to also execute the `GRANT` statement a few lines up in the `if existing_user is not None` block so that we can give already-created users this permission.